app/managers.py

```diff
@@ -0,0 +1,33 @@
+import sqlite3
+
+from models import Actor
+
+class ActorManager:
+    def __init__(self):
+        self._connection = sqlite3.Connection("cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str):
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} (first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+    def all(self):
+        actor_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [
+            Actor(*row) for row in actor_cursor
+        ]
+    def update(self, actor_id: int, first_name: str, last_name: str):
+        self._connection.execute(
+            f"UPDATE {self.table_name} SET first_name=?, last_name=? WHERE id=?",
+            (first_name, last_name, actor_id)
+        )
```

Comment:
The `update` method is missing a call to `self._connection.commit()` after executing the update query. This is necessary to save changes to the database.